examples/Workflow/WorkflowWebApp/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Dapr.Workflow;
+
+// The workflow host is a background service that connects to the sidecar over gRPC
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+// Dapr workflows are registered as part of the service configuration
+builder.Services.AddWorkflow(options =>
+{
+    // Example of registering a "Hello World" workflow function
+    options.RegisterWorkflow<string, string>("HelloWorld", implementation: async (context, input) => 
+    {
+        return await context.CallActivityAsync<string>("SayHello", "World");
+    });
+
+    // Example of registering a "Say Hello" workflow activity function
+    options.RegisterActivity<string, string>("SayHello", implementation: (context, input) =>
+    {
+        return Task.FromResult($"Hello, {input}!");
+    });
```

Comment:
I will let other correct me but if you think of a workflow as defining a graph, activities are the nodes. Both concepts are linked.  https://docs.temporal.io/activities

Reply:
I wouldn't say this is required, just musing since this is a preview/alpha feature:

That was my initial thought, but I guess my question was more do the activities need to be directly related to the workflow, or can they exist outside of one? Should we have some kind of validation that checks that "SayHello" is defiend? Or should we define the activity in the workflow registration/via actual object references?